contract/package.json

```diff
@@ -6,6 +6,7 @@
   "type": "module",
   "scripts": {
     "test": "ava",
+    "build:deployer": "rollup -c rollup.config.mjs",
```

Comment:
Is this project still using rollup?

Reply:
No, it does not but there some tests still using it. Maybe we can clean that later.